bull/google-books-queue/consumer.js

```diff
@@ -66,6 +67,7 @@ GoogleBooksQueue.process((job, done) => {
           "X-archive-meta-licenseurl":
             "https://creativecommons.org/publicdomain/mark/1.0/",
           "X-archive-meta-publisher": publisher.trim(),
+          "X-archive-meta-authors": authors.join().trim(),
```

Comment:
For PDL, we are showing it as "Author", so let's keep the key as X-archive-meta-Author

Reply:
@coderwassananmol 
Fixed
I have corrected the key name